keepassxc-browser/content/totp-field.js

```diff
@@ -57,7 +63,7 @@ kpxcTOTPIcons.isValid = function(field, forced) {
         if (ignoredTypes.some(t => t === field.type)
             || field.offsetWidth < MINIMUM_INPUT_FIELD_WIDTH
             || field.size < 2
-            || (field.maxLength > 0 && (field.maxLength < MIN_INPUT_LENGTH || field.maxLength > MAX_INPUT_LENGTH))
+            || kpxcTOTPIcons.isLengthValid(field)
```

Comment:
These checks seem to be returning true if the field is NOT valid, same for the isLengthValid function. It should be renamed as a result, perhaps isTOTPFieldLengthInvalid

Reply:
I would also maybe just inline the function entirely as it was before. There is no need for a separate function anymore.

Reply:
Check.